DeveloperLabs/MyEmployees/MyEmployees.Updater/ComUpdater/ComBackgroundUpdate.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+// Required for COM background tasks
+using System.Runtime.InteropServices;
+using Windows.ApplicationModel.Background;
+
+// Required for Package and PackageManager API's
+using Windows.ApplicationModel;
+using Windows.Management.Deployment;
+using Windows.Foundation;
+using System.Threading;
+
+namespace MyEmployeesUpdater.ComUpdater
+{
+    [ComVisible(true)]
+    [Guid("095D47F4-030E-4AFF-8963-9CB33D63F682")]
+    // Implements the background updater task using the IBackgroundTask interface
+    public sealed class ComBackgroundUpdate : IBackgroundTask
+    {
+        private volatile int cleanupTask = 0;
+        string newVersion = "2.0.0.0";
+        string inputPackageUri = "c:\\temp\\MyEmployees.Package_2.0.0.0_Test\\MyEmployees.Package_2.0.0.0_x64.msixbundle";
```

Comment:
Hard coded strings are usually not ideal. If this is a sample test location, please comment as to why this specific location is used.